code/modules/events/maintenance_predator_vr.dm

```diff
@@ -71,8 +71,14 @@
 
 /datum/event/maintenance_predator/announce()
 	if(prob(announceProb))
+<<<<<<< HEAD
 		command_announcement.Announce("Unexpected biosignature detected in the maintenance tunnels of [station_name()].", "Lifesign Alert")
 
 //YW Addition: Adding named landmark for events
 /obj/effect/landmark/event_spawn/maintpred
-	name = "maint_pred"
\ No newline at end of file
+	name = "maint_pred"
+||||||| parent of 2ed936c6ee... Merge pull request #9311 from VOREStation/Verkister-patch-5
+		command_announcement.Announce("Unexpected biosignature detected in the maintenance tunnels of [station_name()].", "Lifesign Alert")
+=======
+		command_announcement.Announce("Unexpected biosignature detected in the maintenance tunnels of [station_name()].", "Lifesign Alert")
+>>>>>>> 2ed936c6ee... Merge pull request #9311 from VOREStation/Verkister-patch-5
```

Comment:
#fuk'd

Reply:
I reverted the YW change because I don't think it really needs a landmark where it spawns, dose it? There's no reasoning given in the YW pr.
https://github.com/Yawn-Wider/YWPolarisVore/pull/688